releasenotes/notes/multithreaded-stochastic-swap-6c2f13d7bd566284.yaml

```
@@ -0,0 +1,58 @@
+---
+features:
+  - |
+    The internals of the :class:`.StochasticSwap` algorithm has be reimplmented
+    to be multithreaded and is now written in the
+    `Rust <https://www.rust-lang.org/>`__ programming language instead of Cython.
```

```suggestion
    The internals of the :class:`.StochasticSwap` algorithm have been reimplemented
    to be multithreaded and are now written in the
    `Rust <https://www.rust-lang.org/>`__ programming language instead of Cython.
```

"has/is -> have/are" _might_ just be a BrE/AmE difference - for me "have" matches the plural "internals", but I'm not certain AmE does it the same way.  The other two changes are just typos.